ranger/container/directory.py

```diff
@@ -297,6 +299,7 @@ def inode_filter_func(obj):
         if self.temporary_filter:
```

Comment:
Is this `zf`? And will `zf<CR>` still clear filtering? I'm assuming it won't. Does the filter stack and `zf` even combine?

Reply:
`zf` works like an additional name filter on the stack. They combine and `zf<CR>` doesn't clear the stack, only this single name filter. In the future we could get rid of `temporary_filter`.